src/diagnostics/define_diagnostics.jl

```diff
@@ -843,4 +843,24 @@ function define_diagnostics!(land_model)
             compute_soil_internal_energy!(out, Y, p, t, land_model),
     )
 
+    # SWE
+    add_diagnostic_variable!(
+        short_name = "swe",
+        long_name = "Snow water equivalent",
+        standard_name = "snow_water_equivalent",
+        units = "m",
+        comments = "The height of liquid water if all snow melted",
+        compute! = (out, Y, p, t) ->
+            compute_snow_water_equivalent!(out, Y, p, t, land_model),
+    )
+
+    add_diagnostic_variable!(
```

Comment:
I think it is okay to not say anything for land diagnostics only (but comment if they are also in bucket)?

Reply:
The bucket only has one possible defined surface temperature, so that is what the bucket one returns. For effective surface temperature, I used this definition: https://en.wikipedia.org/wiki/Effective_temperature but I think I implemented it incorrectly.

I am going to remove this new diagnostic from this PR.